.drone.env

```
@@ -1,3 +1,3 @@
 # The test runner source for API tests
-CORE_COMMITID=aacaf7b5cb431713040219c72e91d7883f845652
+CORE_COMMITID= bf1ee0095759cf3132182bc3333194f1fafce162
```

```suggestion
CORE_COMMITID=bf1ee0095759cf3132182bc3333194f1fafce162
```

There is a bonus space here. Probably it does not break anything, but good to remove it.